packages/indexer-agent/src/agent.ts

```diff
@@ -329,33 +329,24 @@ export class Agent {
       sequentialTimerMap(
         { logger, milliseconds: requestIntervalLarge },
         async () => {
-          const deployments = await this.multiNetworks.map(
-            async ({ network }) => {
-              if (
-                this.deploymentManagement === DeploymentManagementMode.AUTO ||
-                network.networkMonitor.poiDisputeMonitoringEnabled()
-              ) {
-                logger.trace('Fetching active deployments')
-                const assignments =
-                  await this.graphNode.subgraphDeploymentsAssignments(
-                    SubgraphStatus.ACTIVE,
-                  )
-                return assignments.map(assignment => assignment.id)
-              } else {
-                logger.info(
-                  "Skipping fetching active deployments fetch since DeploymentManagementMode = 'manual' and POI tracking is disabled",
-                )
-                return []
-              }
-            },
-          )
-          return deployments.values
+          if (this.deploymentManagement === DeploymentManagementMode.AUTO) {
```

Comment:
No more multiNetwork.map here?

Reply:
This was originally to check `network.poiMonitoringEnabled()`. Making this check separate from multiNetworks fixes this check in singleNetworkMode.